src/views/Hooks/ViewHook/index.jsx

```diff
@@ -10,18 +10,32 @@ import createHookQuery from './createHook.graphql';
 import deleteHookQuery from './deleteHook.graphql';
 import updateHookQuery from './updateHook.graphql';
 import triggerHookQuery from './triggerHook.graphql';
+import hookLastFiresQuery from './hookLastFires.graphql';
 
 @hot(module)
 @withApollo
-@graphql(hookQuery, {
-  skip: ({ match: { params } }) => !params.hookId,
-  options: ({ match: { params } }) => ({
-    variables: {
-      hookGroupId: params.hookGroupId,
-      hookId: decodeURIComponent(params.hookId),
-    },
+@compose(
+  graphql(hookLastFiresQuery, {
+    skip: ({ match: { params } }) => !params.hookId,
+    options: ({ match: { params } }) => ({
+      variables: {
+        hookGroupId: params.hookGroupId,
+        hookId: decodeURIComponent(params.hookId),
+      },
+    }),
+    name: 'lastFiresData',
   }),
```

Comment:
No need to create `hookLastFires.graphql`. You should be able to add everything in `hook.graphql`.